### PR TITLE
feat: Add PlaygroundsSubgraphConnector to Llama Hub

### PR DESCRIPTION
**Pull request by Tachikoma000**
### Summary:
Introduced the PlaygroundsSubgraphConnectorToolSpec to Llama Hub, providing blockchain data analysts the ability to easily interface with decentralized subgraphs via the Playgrounds API.

### Details:
- Added the `PlaygroundsSubgraphConnectorToolSpec` class: This serves as the main interface for querying subgraphs on The Graph's decentralized network.
- Provided interface with PlaygroundsAPI. With the PlaygroundsAPI, users are able to query decentralized subgraphs without the need for a crypto wallet or GRTs
- Ensured compatibility and seamless integration with existing tools.
- Provided simple and slightly advanced examples
- Updated README: Included comprehensive guidelines and instructions for users to understand and use the new tool.

### Considerations:
- Future versions might need better error handling and more extensive logging for debugging purposes.
- Integration tests with different subgraphs (hosted and outside The Graph) might be beneficial to ensure a wider coverage of use cases.

### Examples:
Examples can be found in my repo : https://github.com/Tachikoma000/playgrounds_subgraph_connector/blob/main/examples.ipynb

### Testing:
Manually tested the connector with various queries and verified the results against expected outputs. All tests passed successfully.

### CODE REVIEW

**Review by EmanuelCampos — COMMENTED** [review]
nice one, one comment, also can you remove poetry.lock from being checked in?

File: llama_hub/tools/playgrounds_subgraph_connector/requirements.txt [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+SQLAlchemy==2.0.21
+tenacity==8.2.3
+tiktoken==0.5.1
+tqdm==4.66.1
+typing-inspect==0.9.0
+typing_extensions==4.8.0
+tzdata==2023.3
+urllib3==1.26.16
+wrapt==1.15.0
+yarl==1.9.2
```
[the remark this answers is not in the document]
Comment (Tachikoma000) [reply llama_hub/tools/playgrounds_subgraph_connector/requirements.txt]: For sure, Thanks for reviewing!

[the remark this answers is not in the document]
Comment (Tachikoma000) [reply llama_hub/tools/playgrounds_subgraph_connector/requirements.txt]: - Removed poetry.lock
- Removed requirement file. It's not needed; it's safe to assume users will already have Openai and llama_index installed when working with this feature.

[the remark this answers is not in the document]
Comment (EmanuelCampos) [reply llama_hub/tools/playgrounds_subgraph_connector/requirements.txt]: @Tachikoma000 maybe add requests as a requirement can be a good one

[the remark this answers is not in the document]
Comment (Tachikoma000) [reply llama_hub/tools/playgrounds_subgraph_connector/requirements.txt]: > @Tachikoma000 maybe add requests as a requirement can be a good one

Done!

**Comment by Tachikoma000** [comment]
Hi @EmanuelCampos, Quick question: when will the tool be available for viewing/access on the llama hub website?

**Comment by EmanuelCampos** [comment]
@Tachikoma000 

It's already available :)

https://llamahub.ai/l/tools-playgrounds_subgraph_connector

**Pull request merged**